oauth-token-client/src/main/kotlin/com/gxf/utilities/spring/oauth/config/OAuthClientProperties.kt

```diff
@@ -3,18 +3,18 @@
 // SPDX-License-Identifier: Apache-2.0
 package com.gxf.utilities.spring.oauth.config
 
-import com.gxf.utilities.spring.oauth.config.condition.OAuthEnabledCondition
-import org.springframework.beans.factory.annotation.Value
+import com.gxf.utilities.spring.oauth.config.condition.OAuthMsalEnabledCondition
+import org.springframework.boot.context.properties.ConfigurationProperties
 import org.springframework.context.annotation.Conditional
-import org.springframework.context.annotation.Configuration
 import org.springframework.core.io.Resource
 
-@Configuration
-@Conditional(OAuthEnabledCondition::class)
+@Conditional(OAuthMsalEnabledCondition::class)
```

Comment:
How does this properties class work for `FileTokenProvider` with this condition? It does work, but I don't see how...